src/Components/InstanceTypesSelect/index.js

```diff
@@ -43,15 +43,18 @@ const InstanceTypesSelect = () => {
       <SelectOption
         aria-label={'Instance Type item'}
         key={instanceType.id}
-        description={
-          instanceType.cores +
-          ' cores | ' +
-          instanceType.vcpus +
-          ' vCPU | ' +
-          (parseFloat(instanceType.memory) / 1024).toFixed(1) +
-          ' GiB memory | ' +
+        description={''.concat(
```

Comment:
why not use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead of concat?